integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/StormCompatibilitySpec.scala

```diff
@@ -19,52 +19,72 @@ package io.gearpump.integrationtest.checklist
 
 import io.gearpump.integrationtest.TestSpecBase
 import io.gearpump.integrationtest.minicluster.Util
+import io.gearpump.integrationtest.storm.StormClient
 
 /**
  * The test spec checks the compatibility of running Storm applications
  */
 class StormCompatibilitySpec extends TestSpecBase {
 
-  private val STORM_STARTER_JAR = "/opt/gearpump/lib/storm/storm-starter-0.9.5.jar"
+  private val stormClient = new StormClient(cluster, commandLineClient)
 
-  "run storm over gearpump applications" should {
-    "succeed" in {
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    stormClient.start()
+  }
+
+  override def afterAll(): Unit = {
+    stormClient.shutDown()
+    super.afterAll()
+  }
+
+  "Storm over Gearpump" should {
+    "support basic topologies" in {
       // exercise
-      val appId = commandLineClient.submitStormApp(STORM_STARTER_JAR,
-        "storm.starter.ExclamationTopology exclamation")
+      val appId = stormClient.submitStormApp(
+        mainClass = "storm.starter.ExclamationTopology", args = "exclamation")
       Thread.sleep(5000)
 
       // verify
       val actual = expectAppIsRunning(appId, "exclamation")
       Util.retryUntil(restClient.queryStreamingAppDetail(actual.appId).clock > 0)
     }
-  }
 
-  "multilang storm over gearpump" should {
-    "support Python" in {
+    "provide multilang support in python" in {
+      // commented out till python support is added to docker image
+/*      // exercise
+      val appId = stormClient.submitStormApp(config = "",
+        mainClass = "storm.starter.WordCountTopology", args = "wordcount")
+      Thread.sleep(5000)
 
+      // verify
+      val actual = expectAppIsRunning(appId, "wordcount")
+      Util.retryUntil(restClient.queryStreamingAppDetail(actual.appId).clock > 0)*/
     }
-  }
 
-  "storm over gearpump" should {
     "support DRPC" in {
-
-    }
-
-    "support at-least-once semantics with Kafka" in {
-
+      stormClient.submitStormApp(mainClass = "storm.starter.ReachTopology",
+        args = "reach")
+      val drpcClient = stormClient.getDRPCClient
+      drpcClient.execute("reach", "foo.com/blog/1") shouldBe "16"
```

Comment:
The data is not feed explicitly before you submit a storm app. Where these results 16, 14, 0 come from? At least update the test case description. Change "Support DRPC" to something more specific about what is being tested.

BTW: I suggest use `shouldEqual` for values except `shouldBe true` or `shouldBe false`.